src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
Stylesheets with a .scss extension cannot be linked directly in HTML. Instead, you should compile the SCSS to CSS and link the compiled CSS file.